src/30_Advanced/Video_rotate_to_fullscreen_with_hint.html

```diff
@@ -0,0 +1,127 @@
+<!---{
+"preview": "default",
+"default": "preview",
+"skipValidation": true
+}--->
+<!--
+## Introduction
+**This example is experimental. Feature is not valid AMP yet.**
```

Comment:
what does this mean in term of the developer experience: does the developer enable a specific AMP experiment?

Reply:
No, it will just render the document invalid. Added a comment to clarify.